src/Illuminate/Support/At.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Illuminate\Support;
```

Comment:
Would this not be better place inside `Illuminate\Support\DateTime` or something similar?

Reply:
I preferred to make it its own class.